lib/resources/Accounts.js

```diff
@@ -7,6 +7,11 @@ const stripeMethod = StripeResource.method;
 module.exports = StripeResource.extend({
   path: '',
 
+  reject: stripeMethod({
```

Comment:
Flagging that something changed to change the order on some methods. It's not breaking so I'm leaving it but we need to fix codegen to avoid that

Reply:
cc @richardm-stripe as we're merging but this is something that needs to be fixed soon in codegen